<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserAccountWidget</class>
 <widget class="QWidget" name="UserAccountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="title_groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{border:none; border-image: url(:/icons-lib/screen/topbar_long.png);}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>280</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>User Accounts</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>8</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator{width: 25px;height: 25px;}                                               
QCheckBox::indicator:checked {image: url(:/icons-lib/checked_box.png);}
QCheckBox::indicator:unchecked {image: url(:/icons-lib/check_box.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>7</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons-lib/screen/records_icon.png);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/topbar/images/topbar/patients_icon.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_Background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>220</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons-lib/screen/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Login">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>40</y>
     <width>311</width>
     <height>175</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
border:none;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QLabel" name="label_top">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:  url(:/icons-lib/screen/screensection_top.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_topLeft">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/icons-lib/screen/screensection_topleft.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_left">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:  url(:/icons-lib/screen/screensection_left.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_bottomLeft">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/icons-lib/screen/screensection_bottomleft.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_Bottom">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>12</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:  url(:/icons-lib/screen/screensection_bottom.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_topRight">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/icons-lib/screen/screensection_topright.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_right_3">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:  url(:/icons-lib/screen/screensection_right.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_bottomRight">
      <property name="maximumSize">
       <size>
        <width>12</width>
        <height>12</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="logo.qrc">:/icons-lib/screen/screensection_bottomright.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_Login_1">
      <property name="styleSheet">
       <string notr="true">QGroupBox{border:none;
background-image:  url(:/icons-lib/screen/screensection_middle.png);
}
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_add">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>120</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_users">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>287</width>
         <height>110</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>15</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QPushButton" name="pushButton_modify">
       <property name="geometry">
        <rect>
         <x>74</x>
         <y>120</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Modify</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="geometry">
        <rect>
         <x>144</x>
         <y>120</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_close">
       <property name="geometry">
        <rect>
         <x>214</x>
         <y>120</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>[F4] Close</string>
       </property>
       <property name="shortcut">
        <string>F4</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label_Background</zorder>
  <zorder>title_groupBox</zorder>
  <zorder>label_2</zorder>
  <zorder>groupBox_Login</zorder>
 </widget>
 <tabstops>
  <tabstop>checkBox</tabstop>
  <tabstop>tableView_users</tabstop>
  <tabstop>pushButton_add</tabstop>
  <tabstop>pushButton_modify</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>pushButton_close</tabstop>
 </tabstops>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
